spec/sanity/blueprints.js

```diff
@@ -129,9 +123,9 @@ describe('blueprints page', function(){
         components.ui.blueprints.IndexPage.goToBlueprint({'name' : testConf.blueprints.blueprintToRead});
         components.ui.blueprints.BlueprintPage.goToTopology();
         components.ui.blueprints.BlueprintPage.Topology.clickNode(testConf.blueprints.nodeToClick);
-        components.ui.blueprints.BlueprintPage.PropertiesPanel.goToProperties();
-        components.ui.blueprints.BlueprintPage.PropertiesPanel.goToRelationships();
-        components.ui.blueprints.BlueprintPage.PropertiesPanel.close();
+        //components.ui.blueprints.BlueprintPage.PropertiesPanel.goToProperties();
+        //components.ui.blueprints.BlueprintPage.PropertiesPanel.goToRelationships();
+        //components.ui.blueprints.BlueprintPage.PropertiesPanel.close();
 
```

Comment:
You should not remove this lines.
New topology not opening the PropertiesPanel is not how new topology work but a bug which is already fixed and waiting to be merged.